geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ClientUserAuths.java

```diff
@@ -90,28 +120,29 @@ public UserAuthAttributes getUserAuthAttributes(final Long userId) {
   }
 
   @VisibleForTesting
-  protected Collection<Subject> getSubjects() {
-    return Collections.unmodifiableCollection(uniqueIdVsSubject.values());
-  }
-
-  public Subject getSubject(final Long userId) {
-    return uniqueIdVsSubject.get(userId);
-  }
-
-  public void removeSubject(final Long userId) {
-    logger.debug("Subject of {} removed.", userId);
-    removeSubject(uniqueIdVsSubject.remove(userId));
+  @TestOnly
```

Comment:
We should probably have some sort of discussion or tech talk about when `@VisibleForTesting` or `@TestOnly` should be used. Someone needs to own introducing `@TestOnly` and should add a wiki page or other info about when to use these annotations. They should also submit a PR to remove `@VisibleForTesting` and convert all uses to `@TestOnly`. Right now, we're just introducing a mess by using both annotations without any documented guidelines.

Reply:
`@VisibleForTesting` tag was pre-existing, I added the `@TestOnly` because I found out it's only used for test, so `TestOnly` should be more appropriate. I removed the `VisibleForTesting` tag.